### PR TITLE
Update ServiceBus Readme to include an example

### PR DESCRIPTION
**Pull request by devigned**
- Add example from [node-cerulean](https://github.com/noodlefrenzy/node-cerulean/blob/master/lib/index.js) to help users get started.

### CODE REVIEW

**Comment by devigned** [comment]
@noodlefrenzy, @djrosanova showed this to me and I thought it would be awesome to include. I've attributed you as the original source, but if you prefer that it not be included here, we'll not merge the PR.

**Comment by noodlefrenzy** [comment]
@devigned I love it!  Include away :)  Especially since eventually I'll be moving/removing it since node-cerulean is destined for other (nefarious) purposes. (TMI version: Specifically initially I need to wrap blob leases to make them more node-like and let me build something like EventProcessorHost into node-sbus.)

**Comment by devigned** [comment]
@noodlefrenzy thank you for the awesome work! After node-cerulean goes away, I'll drop the link, but leave the attribution.

**Comment by noodlefrenzy** [comment]
@devigned Glad to help! I'll let you know when I make the change, and just move it to an examples directory.  Also, I should probably let @djrosanova know that I accidentally left it running for 3 days and never saw any issues :)

File: lib/services/serviceBus/README.md [hunk 1]
```diff
@@ -14,9 +14,66 @@ npm install azure-sb
 
 ## How to Use
 
-### Authentication
+```node
+var azure = require('azure');
 
-### Create the ServiceBus service
+function checkForMessages(sbService, queueName, callback) {
+    sbService.receiveQueueMessage(queueName, { isPeekLock: true }, function (err, lockedMessage) {
```
Comment (yugangw-msft) [inline lib/services/serviceBus/README.md]: if you still want it to be merged, please get the indent to be 2 spaces

**Comment by yugangw-msft** [comment]
close, and please fix the indent and submit to master when it is ready

**Pull request closed without merging**